apps/nextjs-website/src/app/[productSlug]/tutorials/[...productTutorialPage]/page.tsx

```diff
@@ -64,7 +81,34 @@ const Page = async ({ params }: { params: Params }) => {
   const productSlug = params?.productSlug;
   const tutorialPath = params?.productTutorialPage?.join('/');
 
-  const tutorialProps = await getTutorial(productSlug, [tutorialPath]);
+  const strapiTutorialProps = await getStrapiTutorial(productSlug, [
+    tutorialPath,
+  ]);
+
+  if (strapiTutorialProps) {
+    return (
+      <ProductLayout
+        product={strapiTutorialProps.product}
+        path={strapiTutorialProps.path}
+        // bannerLinks={strapiTutorialProps.bannerLinks} // TODO: refactor bannerLinks
+      >
+        <Box mt={5}>
+          {strapiTutorialProps.title && (
+            <Abstract title={strapiTutorialProps.title} />
+          )}
+          <BlocksRendererClient content={strapiTutorialProps.content} />
```

Comment:
I think we should add an `EContainer` around the `BlocksRendererClient`:
![image](https://github.com/pagopa/developer-portal/assets/39835990/513a3627-d221-4d9a-b256-d8f4b9d7e1f1)